bigbluebutton-html5/imports/ui/components/app/styles.scss

```diff
@@ -120,7 +121,7 @@ $bars-padding: $lg-padding-x - .45rem; // -.45 so user-list and chat title is al
 
 .userlistPad {
   background-color: $color-off-white;
-  width: $border-size-large;
+  width: $userlist-handle-width;
```

Comment:
Why is the userlist changed here?

Reply:
So the userlist resize handle renders as the same size as the chat resize handle when scroll bars are visible.